### PR TITLE
@daimo/contracts: add tokenMetadata

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
Create an autogeneration script to fetch and populate common metadata about the ERC20 token. Saves us a fetch in app or hardcoding at places.

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/DepositScreen.tsx [hunk 1]
```diff
@@ -92,7 +95,8 @@ function OnrampStub() {
     <View style={styles.callout}>
       <TextBody>
         <Octicons name="alert" size={16} color="black" />{" "}
-        <TextBold>Onramp coming soon.</TextBold> You'll be able to buy USDC
+        <TextBold>Onramp coming soon.</TextBold> You'll be able to buy $
+        {tokenMetadata.symbol}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/DepositScreen.tsx:99]: don't need the $, this will render as `$USDC`

Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/DepositScreen.tsx:97]: good catch, fixed!

File: apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx [hunk 2]
```diff
@@ -60,9 +61,9 @@ function IntroPages({ onCreateAccount }: { onCreateAccount: () => void }) {
             technology. Use at your own risk."
         />
         <IntroPage
-          title="USDC"
-          body="Daimo lets you send and receive money using the USDC stablecoin.
-            1 USDC is $1. Learn how it works here."
+          title={tokenMetadata.symbol}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx:64]: hmmmm.

if someone forks Daimo, they should write their own onboarding copy.

File: packages/daimo-contract/src/tokenMetadata.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Generated by script/fetchTokenMetadata.ts
+
+export const tokenMetadata = {
+  address: "0x1B85deDe8178E18CdE599B4C9d913534553C3dBf",
+  decimals: 6,
+  symbol: "USDC",
+  name: "testUSDC",
+  totalSupply: "1000000000000",
```
Comment (dcposch) [inline packages/daimo-contract/src/tokenMetadata.ts:8]: tbh this is a tiny amount of metadata

for a huge decision (which coin Daimo runs on)

idk if codegen is necessary for that

(not a big deal though; checking it in to `daimo-contract` so we can import everywhere else lgreattm)

**Review by dcposch — APPROVED** [review]
LGTM

File: apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx (hunk 2 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx:64]: fine here

might add a forking guide in future

File: packages/daimo-contract/src/tokenMetadata.ts (hunk 3 above)
Comment (nalinbhardwaj) [reply packages/daimo-contract/src/tokenMetadata.ts:8]: yeah, good point!

**Pull request merged**